final_results/result_processing_wrapper.sh

```diff
@@ -19,7 +19,7 @@ SCRIPT_DIR=$REPO_DIR/final_results # todo, move them both: s$REPO_DIR/cripts
 JDK_ver=$1
 benchmark=$2
 if [[ $benchmark == "" ]];then
-  benchmark="DACAPO J2DBENCH JMH RADARGUNs1" # TODO, ADD ALL
+  benchmark="DACAPO J2DBENCH JMH RADARGUNs1 RADARGUNs3 SPECJBB"
```

Comment:
I htink this filechanges have nothng to do with plots, and should be in different commit.

Reply:
yes